### PR TITLE
ARTEMIS-602 Fix REST examples

### PR DESCRIPTION
**Pull request by jbertram**

### CODE REVIEW

**Comment by jbertram** [comment]
The PR test build fails with:

[ERROR] Failed to execute goal on project artemis-web: Could not resolve dependencies for project org.apache.activemq:artemis-web:jar:1.4.0-SNAPSHOT: Could not find artifact org.eclipse.jetty.aggregate:jetty-all:jar:9.3.10.v20160621 in central (http://repo.maven.apache.org/maven2) -> [Help 1]

However, the dependency is there at http://repo.maven.apache.org/maven2/org/eclipse/jetty/aggregate/jetty-all/9.3.10.v20160621/ so I don't understand what the problem is.

**Comment by jbertram** [comment]
I sorted the dependency issue out, and I also ran the PR check on my local machine since the Apache CI seems to be flaky right now. It passed.

File: examples/features/standard/rest/dup-send/README.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 System Requirements:
-You will need JDK 1.6 and Maven to run this example.  This example has been tested with Maven 2.2.1.  It may or may not work 
+You will need JDK 1.8 and Maven to run this example.  This example has been tested with Maven 3.3.3.  It may or may not work
```
Comment (clebertsuconic) [inline examples/features/standard/rest/dup-send/README.txt]: 1.7 ?

File: examples/features/standard/rest/javascript-chat/README.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 System Requirements:
-You will need JDK 1.6, Maven, and a browser to run this example.  This example has been tested with Maven 2.2.1.  It may or may not work
```
Comment (clebertsuconic) [inline examples/features/standard/rest/javascript-chat/README.txt]: 1.7?

File: examples/features/standard/rest/dup-send/README.txt (hunk 1 above)
Comment (jbertram) [inline examples/features/standard/rest/dup-send/README.txt]: Are we stuck on 1.7?  It's been EOL for over a year now.

Comment (clebertsuconic) [inline examples/features/standard/rest/dup-send/README.txt]: We can probably leave 1.8 on the docs, and still let the build to work with 1.7.

**Pull request closed without merging**